static/alert.html

```diff
@@ -104,6 +104,7 @@
                         <button class="btn" id="save-alert-btn" type="submit" tabindex="10">Save</button>
                     </div>
                     <div class="add-alert-form">
+                        <h1 class="rulename"></h1>
```

Comment:
Make the CSS for Rulename as 
```     
    font-size: 18px;
    font-weight: 700;
    margin-bottom: 20px;
    margin-left: 5px;
  ```
Also, set the display to none for the creation time and show it only during edit time.